Makefile

```diff
@@ -0,0 +1,22 @@
+include k8s.mk
```

Comment:
This file seems to be missing

Reply:
correct, fixed in [b0c790d](https://github.com/neuro-inc/mlops-spark-operator/pull/1/commits/b0c790d21249f3c4a713ed7c02a0ffd37dd3fc6e)